### PR TITLE
Use v8 instead of JSC + native SVG support

### PR DESCRIPTION
**Pull request by brunobar79**
This PR was an attempt to fix the crash happening when going to settings, which took me to try `react-native-v8` instead of JSC.  Performance is way better, [as seen here](https://docs.google.com/spreadsheets/d/1uce3WZ9IaAEUu6Owj3eXEuZb25PDi6ZcgUVV2i500S0/edit#gid=469794458).

After that, I realized the crash was still there and it wasn't JSC fault, so I kept digging until I found it was being caused by `react-native-remote-svg` while rendering the webviews outside of the viewport (basically the tokens elements  that were below the fold). Since the webview wasn't visible android was attempting to remove it from the view stack when other view was displayed on top (ex. going to settings), causing a memory leak that would lead to a crash, and it resulted to be [our # 1 crash](https://fabric.io/metamask/android/apps/io.metamask/issues?status=all&event_type=crash&time=last-thirty-days&subFilter=state&state=open&build%5B0%5D=top-builds)!

That has been fixed by removing the dependency from `react-native-remote-svg` and use `react-native-svg` instead, which now supports remote svgs too and it renders them natively.

I've wrapped into a component `RemoteImage` that we should use to load external images.

TL;DR: The app is way faster and we fixed the main crash.

### CODE REVIEW

**Review by ibrahimtaveras00 — APPROVED** [review]
🔥 🔥 🔥 🔥 This is amazing, huge difference 

Crash no longer happens on android, and performance is notably better, double thumbs up here, QA PASSED

**Review by estebanmino — APPROVED** [review]
👏 👏
